yarn.lock

```diff
@@ -12100,6 +12133,11 @@ vary@~1.1.2:
   resolved "https://registry.yarnpkg.com/vary/-/vary-1.1.2.tgz#2299f02c6ded30d4a5961b0b9f74524a18f634fc"
   integrity sha1-IpnwLG3tMNSllhsLn3RSShj2NPw=
 
+velocity-animate@^2.0.6:
```

Comment:
Uh oh, it came back from the dead... Maybe try editing the `yarn.lock` to delete `matrix-react-sdk` block, save, re-run `yarn install`, and see if that helps? (Yarn is generally confused by our use of Git-based dependencies).